index.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const products = require('./productRouter.js');
+const cart = require('./cartRouter.js');
+const wishlist = require('./wishlistRouter.js');
+const {error404} = require('./error-handler');
+const mongoose = require('mongoose');
+const cors = require('cors');
+const PORT = process.env.PORT || 3000;
+
+const app = express();
+app.use(bodyParser.json());
+app.use(cors());
+
+mongoose.connect(`mongodb+srv://vkecomm:${process.env.MONGO_DB}@thecameracompanyvk.qkv6e.mongodb.net/inventory?retryWrites=true&w=majority`, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+})
+.then(() => console.log("mongoose success"))
+.catch(err => console.error("mongoose failed", err))
+
+
+myLogger = (req, res, next) => {
```

Comment:
Two separate scirts in package.json:
1. For production: "dev": "NODE_ENV=development node server.js"
2. For development: "build": "NODE_ENV=production node server.js"

```javascript
if (process.env.NODE_ENV === development ) {
   myLogger = (req, res, next) => { 
   
   }
}
```